bin/setup_dot1x

```diff
@@ -3,14 +3,14 @@
 ROOT=$(dirname $0)/..
 cd $ROOT
 
-sudo rm -rf inst/faucet/faucet_dot1x
-mkdir -p inst/faucet/faucet_dot1x
+#sudo rm -rf inst/faucet/daq-faucet-dot1x
```

Comment:
erm?  Remove dead code unless there's a specific reason... or leave it in?

Reply:
done